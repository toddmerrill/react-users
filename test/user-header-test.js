import React from 'react';
import {UserHeader} from '../src/components/Users.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <UserHeader />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
