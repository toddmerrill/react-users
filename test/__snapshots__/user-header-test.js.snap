// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="listHeader"
>
  <h2
    className="oneLine"
  >
    Users
  </h2>
  <span
    className="rightButtons"
  >
    <a
      className="nohover"
      onClick={[Function]}
    >
      <span
        className="btn-circle btn-ok"
      />
    </a>
    <a
      className="nohover"
      onClick={[Function]}
    >
      <span
        className="btn-circle btn-error"
      />
    </a>
  </span>
</div>
`;
